# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/generic/nsFloatManager.h
# Commit: f2c924263ffe
# Full Hash: f2c924263ffece6d0b30e6a52dfd32792c5bf5ab
# Author: Frédéric Wang <fwang@igalia.com>
# Date: 2024-01-16 16:37:21
# Regressor Bug: 1765615
# File Overlap Count: 1
# Description:
#   Bug 1765615 - Introduce NS_BLOCK_DYNAMIC_BFC state bit. r=layout-reviewers,emilio
#   
#   After D198523, NS_BLOCK_FLOAT_MGR and NS_BLOCK_MARGIN_ROOT are always
#   set simultaneously when initiating a frame, so we can merge them into a
#   single NS_BLOCK_STATIC_BFC flag. The freed bit is used to define a new
# ==============================================================================

diff -r d72041117125 -r f2c924263ffe layout/generic/nsFloatManager.h
--- a/layout/generic/nsFloatManager.h	Tue Jan 16 07:00:26 2024 +0000
+++ b/layout/generic/nsFloatManager.h	Tue Jan 16 07:58:47 2024 +0000
@@ -70,7 +70,7 @@
 /**
  * nsFloatManager is responsible for implementing CSS's rules for
  * positioning floats. An nsFloatManager object is created during reflow for
- * any block with NS_BLOCK_FLOAT_MGR. During reflow, the float manager for
+ * any block with NS_BLOCK_BFC_STATE_BITS. During reflow, the float manager for
  * the nearest such ancestor block is found in ReflowInput::mFloatManager.
  *
  * According to the line-relative mappings in CSS Writing Modes spec [1],